content/sensu-go/6.9/operations/control-access/ad-auth.md

```diff
@@ -428,7 +432,8 @@ servers:
 
 | allowed_groups |   |
 -------------|------
-description  | An array of allowed AD group strings to include in the tokenized identity claim. Use to specify which groups to encode in the authentication provider's JSON Web Token (JWT) when the authenticated AD user is a member of many groups and the tokenized identity claim would be too large for correct web client operation.
+description  | An array of allowed AD group strings to include in the tokenized identity claim. Use to specify which groups to encode in the authentication provider's JSON Web Token (JWT) when the authenticated AD user is a member of many groups and the tokenized identity claim would be too large for correct web client operation.{{% notice note %}}**NOTE**: Allowed group names are case-sensitive and must exactly match the group names the authentication provider returns to the Sensu backend.
+{{% /notice %}}
 required     | false
 type         | Array
 example      | {{< language-toggle >}}
```

Comment:
Are the example names we use in the docs valid? I noticed in your Slack discussion, the examples use sentence case with underscores. I thought perhaps they should change from `sensu-viewers` to `Sensu_Viewers` for example (see lines 441-443 below to see what I mean).

Reply:
I think the example names are valid.  The examples in the slack discussion were from the customer issue and are also valid.  I believe the change you're suggesting would 'underscore' the point about case sensitivity.